astropy/utils/console.py

```diff
@@ -860,7 +860,7 @@ class Spinner:
 
         with Spinner("Reticulating splines", "green") as s:
             for item in enumerate(items):
-                s.next()
+                next(s) # or s.update()
```

Comment:
I would almost rather just document it as `s.update()` since I think that's less confusing and surprising to beginners (who might not understand where `next()` came from).